frontend/javascripts/oxalis/view/context_menu.js

```diff
@@ -177,37 +191,65 @@ function NodeContextMenuOptions({
 }
 
 function NoNodeContextMenuOptions({
+  isSkeletonToolActive,
   hideNodeContextMenu,
   globalPosition,
   viewport,
   createTree,
+  cellIdAtPosition,
 }: NoNodeContextMenuProps) {
+  const skeletonActions = [
+    <Menu.Item
+      className="node-context-menu-item"
+      key="create-node"
+      onClick={() => setWaypoint(globalPosition, viewport, false)}
+    >
+      Create Node here
+    </Menu.Item>,
+    <Menu.Item
+      className="node-context-menu-item"
+      key="create-node-with-tree"
+      onClick={() => {
+        createTree();
+        setWaypoint(globalPosition, viewport, false);
+      }}
+    >
+      Create new Tree here
+    </Menu.Item>,
+  ];
+  const nonSkeletonActions = [
+    <Menu.Item
+      className="node-context-menu-item"
+      key="select-cell"
+      onClick={() => handlePickCellFromGlobalPosition(globalPosition)}
+    >
+      Select Cell ({cellIdAtPosition})
+    </Menu.Item>,
+
+    <Menu.Item
+      className="node-context-menu-item"
+      key="fill-cell"
+      onClick={() => handleFloodFillFromGlobalPosition(globalPosition, viewport)}
+    >
+      Fill Cell (flood-fill region)
+    </Menu.Item>,
```

Comment:
Shouldn't these options be restricted to annotations with a segmentation layer? Or are they, and I don't see it?

Reply:
You're right! Fixed it in the meantime :)